templates/command/InvokableCommand.tpl.php

```diff
@@ -0,0 +1,32 @@
+<?= "<?php\n"; ?>
+
+namespace <?= $namespace; ?>;
+
+<?= $use_statements; ?>
+
+#[AsCommand(
+    name: '<?= $command_name; ?>',
+    description: 'Add a short description for your command',
+)]
+class <?= $class_name; ?>
+{
+    public function __invoke(
+        SymfonyStyle                                                  $symfonyStyle,
+        #[Argument('Required argument description')] string           $requiredArg,
+        #[Argument('Optional nullable argument description')] ?string $optionalNullableArg,
+        #[Argument('Optional non-nullable argument description')] int              $optionalNonNullableArg = 1,
+        #[Option('Option description')] string                        $option = "defaultOptionValue",
+        #[Option('Nullable option description')] ?string              $nullableOption = null,
```

Comment:
```suggestion
        SymfonyStyle $io,
        #[Argument('Required argument description')] string $requiredArg,
        #[Argument('Optional nullable argument description')] ?string $optionalNullableArg,
        #[Argument('Optional non-nullable argument description')] int $optionalNonNullableArg = 1,
        #[Option('Option description')] string $option = "defaultOptionValue",
        #[Option('Nullable option description')] ?string $nullableOption = null,
```

Reply:
Anyway, I don't think we need so much examples, let's use something lighter instead: 
```suggestion
        SymfonyStyle $io,
        #[Argument] string $name,
        #[Option] bool $enable = false,
```

Reply:
@Kocal I agree. I did this, because I noticed the documentation is not complete yet. So I added more examples to make it clearer what different options there are. For example the optional nullable argument is not intuitive at all if you ask me. It is nullable but does not need a default value. Usually I would expect an error because I did not pass the argument. In this case, Symfony simply passes null if the argument is not passed in the console. Which does not seem obvious to me.

Reply:
I see what you mean, but then, maybe this is more a documentation issue that.. should be fixed on the documentation itself? ;)

Reply:
@Kocal fair point. I reduced the number of parameters in the template 👍